lms/static/sass/views/_shoppingcart.scss

```diff
@@ -637,390 +532,480 @@
-      padding: 20px;
-      border-radius: 4px;
-      margin-bottom: ($baseline*0.75);
-      min-height: 240px;
-      &:nth-child(even){
-        margin-left: 30px;
-      }
-      .data-group{
+
+    .col-half {
+      @include float(left);
+      @include border-radius(4px);
       margin-bottom: ($baseline*0.75);
-        label{
-        display: block;
-        font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-        font-size: 16px;
-        font-style: normal;
-        font-weight: 400;
-        color: $dark-gray2;
+      background: $light-gray1;
+      padding: $baseline;
+      width: 45%;
+      min-height: ($baseline*12);
+
+      // CASE: index of child is even
+      &:nth-child(even) {
+        @include margin-right($baseline*1.5);
       }
-      input{width: 100%;margin-bottom: ($baseline/4);}
-      &:nth-child(4n){
-        margin-right: 0;
+
+      .data-group {
+        margin-bottom: ($baseline*0.75);
+
+        label {
+          @extend %t-icon5;
+          @extend %t-regular;
+          display: block;
+          color: $dark-gray2;
+          font-family: $sans-serif;
+          font-style: normal;
+        }
+
+        input {
+          margin-bottom: ($baseline/4);
+          width: 100%;
+        }
+
+        // CASE: index of child is a multiple of 4
+        &:nth-child(4n) {
+          @include margin-right(0);
+        }
       }
     }
-    }
   }
+
   .error-text{
-    color: $red1;
-    font-size: 12px;
+    @extend %t-icon7;
     display: block;
     padding-bottom: 0;
+    color: $red1;
   }
-  .gray-bg{
+
+  .gray-bg {
+    @include border-radius(3px);
+    @include padding($baseline, $baseline, $baseline, $baseline*1.5);
+    margin: $baseline 0;
     background: $light-gray1;
-    border-radius: 3px;
-    padding: 20px 20px 20px 30px;
-    margin: 20px 0;
     overflow: hidden;
-    .message-left{
+
+    .message-left {
+      @include float(left);
+      @include line-height(16.22);
       width: 100%;
-      float: left;
-      line-height: 24px;
       color: $dark-gray1;
-      b{
+
+      b {
         text-transform: capitalize;
       }
-      a.blue{
-        margin:0 0 0 20px;
-        i{
-          margin-left: ($baseline/2);
+
+      a.blue {
+        @include margin(0, 0, 0, $baseline);
+
+        i {
+          @include margin-left($baseline/2);
         }
       }
+
       .mt-7 {
         display: block;
-        margin-top: 7px;
+        margin-top: ($baseline*0.35);
       }
     }
   }
-  .bordered-bar{
-    border-bottom: 2px solid $light-gray1;
-    border-top: 2px solid $light-gray1;
+
+  .bordered-bar {
     margin-bottom: $baseline;
-    padding: 20px;
-    h2{
-      color: $dark-gray1;
-      font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-      font-weight: bold;
+    border-top: $discount-border;
+    border-bottom: $discount-border;
+    padding: $baseline;
+
+    h2 {
+      @include font-size(17);
+      @extend %t-strong;
```

Comment:
using %t-title5 cleans this rule group up a bit. (removes font-size and font-family rules)